application/src/main/java/org/togetherjava/tjbot/commands/help/AskCommand.java

```diff
@@ -146,6 +132,40 @@ private boolean isUserOnCooldown(User user) {
             .isPresent();
     }
 
+    private void sendCooldownResponse(SlashCommandInteractionEvent event) {
+        User user = event.getUser();
+
+        String message =
+                """
+                        Sorry, you can only create a single help thread every 5 minutes. Please use your existing thread %s instead.
+                        If you made a typo or similar, you can adjust the title using the command %s and the category with %s :ok_hand:""";
+
+        RestAction<String> changeTitle = MessageUtils.mentionGuildSlashCommand(event.getGuild(),
+                HelpThreadCommand.COMMAND_NAME, HelpThreadCommand.CHANGE_SUBCOMMAND_GROUP,
+                HelpThreadCommand.CHANGE_TITLE_SUBCOMMAND);
+        RestAction<String> changeCategory = MessageUtils.mentionGuildSlashCommand(event.getGuild(),
+                HelpThreadCommand.COMMAND_NAME, HelpThreadCommand.CHANGE_SUBCOMMAND_GROUP,
+                HelpThreadCommand.CHANGE_CATEGORY_OPTION);
+        long lastCreatedThreadId = database
+            .read(context -> context.selectFrom(HELP_THREADS)
+                .where(HELP_THREADS.AUTHOR_ID.eq(user.getIdLong()))
+                .orderBy(HELP_THREADS.CREATED_AT.desc())
+                .fetch())
+            .get(0)
```

Comment:
use `fetchOne()` instead.

Reply:
```
org.togetherjava.tjbot.db.DatabaseException: org.jooq.exception.TooManyRowsException: Cursor returned more than one result
	at org.togetherjava.tjbot.db.Database.read(Database.java:92) ~[database-1.0-SNAPSHOT.jar:?]
	at org.togetherjava.tjbot.commands.help.AskCommand.sendCooldownResponse(AskCommand.java:146) ~[main/:?]
	at org.togetherjava.tjbot.commands.help.AskCommand.onSlashCommand(AskCommand.java:98) ~[main/:?]
```
 getting this error when using fetchOne going back to what it was before

Reply:
@Zabuzard

Reply:
because u have to limit it to 1 in the query.

Reply:
done